data/modules/SearchRescue/SearchRescue.lua

```diff
@@ -613,7 +613,14 @@ local createTargetShipParameters = function (flavour)
 	---- loading drive, weapons etc.
 	if flavour.id == 1 or flavour.id == 6 then
 		for i,shipdef in pairs(shipdefs) do
-			if shipdef.capacity / 10 < 1 then shipdefs[i] = nil end
+
+			-- get mass of hyperdrive if this ship has a default drive
+			-- if no default drive only 4t will have to be assumed for lowest tech drive
+			-- higher tech drives will only be fitted later if capacity is huge
+			local drive_mass = 4
+			local default_drive = Equipment.hyperspace['hyperdrive_'..tostring(shipdef.hyperdriveClass)]
```

Comment:
technically, I can imagine we sometimes in the future rebalance ships and hyperdrives, so hard coding `drive_mass` might only work up til then. Is it possible to have query / sort all hyperdrives, and then get the smallest number out? Would be a more stable solution

Reply:
Okay, I understand your point. I'll query for the lowest mass engine instead of hardcoding its current mass.